lib/wikidata_summary_parser.rb

```diff
@@ -140,6 +142,10 @@ def created_item?
     @summary.include? 'wbeditentity-create'
   end
 
+  def created_lexeme_item?
+    @summary.include? 'wbeditentity-create-lexeme'
```

Comment:
This string includes the substring `wbeditentity-create`, so we've already been miscounting lexeme creations as item creation. With this update, now a lexeme creation gets counted as both item creation and lexeme creation. I guess the matcher for `created_item?` needs to be updated to make sure it excludes lexemes.

Reply:
Ooups. You are right. I will update the `created_item?` method.